src/octoprint/templates/tabs/terminal.jinja2

```diff
@@ -13,7 +13,8 @@
         <span data-bind="visible: !autoscrollEnabled() && enableFancyFunctionality()" style="display: none">(<a href="javascript:void(0)" data-bind="click: scrollToEnd">{{ _("Scroll to end") }}</a>)</span>
     </small>
     <small class="pull-right">
-        <a href="javascript:void(0)" data-bind="click: copyAll"><i class="fa fa-copy" title="{{ _('Copy all') }}"></i> {{ _('Copy all') }}</a>
+        <a href="javascript:void(0)" data-bind="click: copyAll"><i class="fa fa-copy" title="{{ _('Copy all') }}"></i> {{ _('Copy all') }}</a><br>
+        <a href="javascript:void(0)" data-bind="click: clearLogWidow"><i class="fa fa-copy" title="{{ _('Clear log widow') }}"></i> {{ _('Clear log widow') }}</a>
```

Comment:
Typo ;) "widow" -> "window". I'd actually turn it into just "Clear all" though, at least for the link text itself and leave the title as is.

Reply:
Totally - no idea how on earth I didn't spot that - been clicking that button many many times and just skipepd over it! Fixed and changed link to ClearAllLogs